modules/tensor_mechanics/tests/mohr_coulomb/tests

```diff
@@ -126,6 +133,13 @@
     rel_err = 1.0E-5
     abs_zero = 1.0E-3
   [../]
+  [./uni_axial1_small_strain]
+    type = 'CSVDiff'
+    input = 'uni_axial1_small_strain.i'
+    csvdiff = 'uni_axial1.csv'
+    rel_err = 1.0E-5
+    abs_zero = 1.0E-3
```

Comment:
OK @friedmud , we'll have to agree to disagree on that.  I almost always use "abs_tol" in my real simulations as i need to ensure that things have converged to a suitable solution.  Using "rel_tol" means the final result depends upon any crazy ICs/BCs that happen to make the initial residual huge or tiny, which is not appropriate in my real simulations.